### PR TITLE
Fixing Typing Errors

### PR DESCRIPTION
**Pull request by amragaey**
Referring to #19 
This is the proposed solution.

### CODE REVIEW

**Review by ErikSchierboom — CHANGES_REQUESTED** [review]
First of all, thanks for reporting the issue and even submitting a PR to fix it! I have one request: instead of commenting the unnecessary lines, could you remove them?

**Comment by amragaey** [comment]
I removed the unnecessary lines, hope you could merge now peacefully.

Thanks!

File: dist/knockout.d.ts [hunk 1]
```diff
@@ -35,7 +29,7 @@ interface KnockoutObservableArrayFunctions<T> {
     sort(compareFunction: (left: T, right: T) => number): void;
 
     // Ko specific
-    [key: string]: KnockoutBindingHandler;
+    // [key: string]: KnockoutBindingHandler;
```
Comment (ErikSchierboom) [inline dist/knockout.d.ts]: Could you perhaps also remove this comment?

**Review by ErikSchierboom — CHANGES_REQUESTED** [review]

**Comment by amragaey** [comment]
Thanks for your attention, check now!

**Review by ErikSchierboom — APPROVED** [review]
Great work! Thanks. :tada:

**Pull request merged**